sql_scripts/3_cities.sql

```diff
@@ -0,0 +1,567 @@
+CREATE TABLE IF NOT EXISTS cities
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    state_id INTEGER  NOT NULL 
+        references states (id)
```

Comment:
state_id refference states.id which is just autoincrement field and this cannot be valid relation. Make states.id values insertable as was already mentioned in the comments left in sql_scripts/2_states